docs/sources/enterprise/access-control/permissions.md

```diff
@@ -21,55 +21,57 @@ scope
-users.quotas:update | global:users:* | Allows to update user quotas.
-org.users.read | users:* | Allows to get user profiles within the organization.
-org.users.add | users:* | Allows to add users to the organization.
-org.users.remove | users:* | Allows to remove users from the organization.
-org.users.role:update | users:* | Allows to update users organization role for the assigned organization.
-ldap.user:read | n/a | Allows to read LDAP users.
-ldap.user:sync | n/a | Allows to sync LDAP users.
-ldap.status:read | n/a | Allows to check LDAP status.
+roles:list | roles:* | List available roles without permissions.
+roles:read | roles:* | Read a specific role with it's permissions.
+roles:write | permissions:delegate | Create or update a custom role.
+roles:delete | permissions:delegate | Delete a custom role.
+roles.builtin:list | roles:* | List built-in role assignments.
+roles.builtin:add | permissions:delegate | Create a built-in role assignment.
+roles.builtin:remove | permissions:delegate | Delete a built-in role assignment.
+reports.admin:create | reports:* | Create reports.
+reports.admin:write | reports:* | Update reports.
+reports:delete | reports:* | Delete reports.
+reports:read | reports:* | List all available reports or get a specific report.
+reports:send | reports:* | Send a report email.
+reports.settings:write | n/a | Update report settings.
+reports.settings:read | n/a | Read report settings.
+provisioning:reload | service:access-control | Reload provisioning files.
+users:read | global:users:* | Read or search user profiles.
+users:write | global:users:* | Update a user’s profile.
+users.teams:read | global:users:* | Read a user’s teams.
+users.authtoken:list | global:users:* | List authentication tokens that are assigned to a user.
+users.authtoken:update | global:users:* | Update authentication tokens that are assigned to a user.
+users.password:update | global:users:* | Update a user’s password.
+users:delete | global:users:* | Delete a user.
+users:create | n/a | Create a user.
+users:enable | global:users:* | Enable a user.
+users:disable | global:users:* | Disable a user.
+users.permissions:update | global:users:* | Update a user’s organization-level permissions.
+users:logout | global:users:* | Log out a user.
+users.quotas:list | global:users:* | List a user’s quotas.
+users.quotas:update | global:users:* | Update a user’s quotas.
+org.users.read | users:* | Get user profiles within an organization.
+org.users.add | users:* | Add a user to an organization.
+org.users.remove | users:* | Remove a user from an organization.
+org.users.role:update | users:* | Update the organization role (`Viewer`, `Editor`, `Admin`) for an organization.
+ldap.user:read | n/a | Get a user via LDAP.
+ldap.user:sync | n/a | Sync a user via LDAP.
+ldap.status:read | n/a | Verify the LDAP servers’ availability.
+status:accesscontrol | service:access-control | Get access-control enabled status.
```

Comment:
Added this one. Need to double check we are ok to communicate about it. @vtorosyan ?

Reply:
We did advertise the enpoint in docs/source/http_api/access_control.md so I assume this is ok :)

Reply:
yes this is needed